deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -419,6 +419,23 @@ spec:
             dependencies:
               description: External dependencies needed by the Infinispan cluster
               properties:
+                artifacts:
+                  items:
+                    properties:
+                      type:
+                        description: ExternalArtifactType external artifact file type
```

Comment:
Can we improve this description? This will appear in OCP console

Reply:
Yes, @Crumby 
Probably we need to brake golang comment convention and add more human readable notes to all our types in the CRD

Reply:
Done, thanks